9_report/src/fig_error_v_leadtime.R

```diff
@@ -86,6 +93,9 @@ fig_error_v_leadtime <- function(fig_ind, config_fig_yml, preds_ind, agg_nwis_in
                      xlab('Lead Time (days)') +
                      theme(legend.position = 'none',
                            axis.title.y = element_blank(),
+                           axis.text = element_text(size = 15),
+                           strip.text = element_text(size = 15),
+                           axis.title = element_text(size = 15),
```

Comment:
i'm not worried about this for now, but i wonder if we can create a shared theme at some point that takes out some need for repeating this text in many places. let's just keep an eye out and decide later whether that'd be useful.

Reply:
yup we can add text style / font size / fig size to `config.fig.yml`

Reply:
that's probably plenty for this poster. but what i meant was something like
```r
iemss_theme <- theme(
  axis.text = element_text(size = 15),
  strip.text = element_text(size = 15),
  axis.title = element_text(size = 15)
)
```
which could be defined in its own shared file as an R object and could be tacked on as `g <-
 ggplot(...) + ... + iemss_theme` within each of the figure code files. this saves the repeated calls to `element_text`.